core/embed/firmware/memory_T.ld

```diff
@@ -22,8 +23,9 @@ data_size = SIZEOF(.data);
 ccmram_start = ORIGIN(CCMRAM);
 ccmram_end = ORIGIN(CCMRAM) + LENGTH(CCMRAM);
 
-/* IMAGE_HEADER_SIZE is 0x400, this is for interaction-less firmware update start */
-firmware_header_start = ccmram_end - 0x400;
+/* reserve 256 bytes for bootloader arguments */
+boot_args_start = ORIGIN(BOOT_ARGS);
```

Comment:
i think we should also take care and clear boot args at the start of firmware/prodtest/reflash in respective startup.s files.

Reply:
I've added clearing of boot_arg, see 5eb2365acd075c7d95cb4b3a7da6ba29c5ce1f64